app/models/episode.rb

```diff
@@ -13,25 +14,20 @@ class Episode < ApplicationRecord
 
   attr_accessor :strict_validations
 
-  serialize :categories, JSON
-  serialize :keywords, JSON
-
   acts_as_paranoid
 
+  serialize :categories, JSON
+  serialize :keywords, JSON
   serialize :overrides, HashSerializer
 
   belongs_to :podcast, -> { with_deleted }, touch: true
-
-  has_many :images,
-    -> { order("created_at DESC") },
-    class_name: "EpisodeImage", autosave: true, dependent: :destroy
-
-  has_many :contents,
-    -> { order("position ASC, created_at DESC") },
-    autosave: true, dependent: :destroy
+  has_many :contents, -> { order("position ASC, created_at DESC") }, autosave: true, dependent: :destroy
+  has_many :images, -> { order("created_at DESC") }, class_name: "EpisodeImage", autosave: true, dependent: :destroy
+  has_one :uncut, -> { order("created_at DESC") }, autosave: true, dependent: :destroy
```

Comment:
It doesn't have any overlap with other terms, which is mainly what I was after.

Naming gets a bit odd, as `MediaResource` is our parent model for these.  I was trying to avoid using "media" in the name.  Also considered `Unsegmented` - but this one really jumps out at you, you know?  No mistaking what it is.